paymentsheet/src/test/java/com/stripe/android/customersheet/CustomerSheetViewModelTest.kt

```diff
@@ -864,10 +869,10 @@ class CustomerSheetViewModelTest {
             workContext = testDispatcher,
             isGooglePayAvailable = false,
             customerPaymentMethods = listOf(),
-            customerAdapter = FakeCustomerAdapter(
+            intentDataSource = FakeCustomerSheetIntentDataSource(
                 canCreateSetupIntents = true,
-                onSetupIntentClientSecretForCustomerAttach = {
-                    CustomerAdapter.Result.success("invalid setup intent")
+                onRetrieveSetupIntentClientSecret = {
+                    CustomerSheetDataResult.success("invalid setup intent")
```

Comment:
not necessary to fix in this PR, but should this set up intent be something like si_12345? I feel like that's more consistent with what we usually do for PMs + PIs

Reply:
or seti_123 which is used below

Reply:
Will follow-up!